rpcs3/headless_application.cpp

```diff
@@ -67,16 +67,11 @@ void headless_application::InitializeCallbacks()
 	{
 		switch (const video_renderer type = g_cfg.video.renderer)
 		{
-		case video_renderer::null:
-		{
-			g_fxo->init<rsx::thread, named_thread<NullGSRender>>();
-			break;
-		}
 		case video_renderer::opengl:
 		case video_renderer::vulkan:
 		{
-			fmt::throw_exception("Headless mode can only be used with the %s video renderer. Current renderer: %s", video_renderer::null, type);
-			[[fallthrough]];
+			g_fxo->init<rsx::thread, named_thread<NullGSRender>>();
```

Comment:
This is where it gets complicated. We do have a headless mode for vulkan with offscreen rendering, its just not hooked up here. But it should be in the end at least, because we have plans for having this mode used as a test bot for which we get graphical output.
I'm not sure how this will work, we'll end up reintroducing null as a renderer option here for non-graphical tests?